config/environments/production.rb

```diff
@@ -51,15 +48,15 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
Intentional?

Reply:
Probably added by the rails bot - will change this one back